drivers/python/rethinkdb/net_twisted.py

```diff
@@ -156,67 +141,40 @@ def clientConnectionFailed(self, connector, reason):
         pass
 
 class CursorItems(DeferredQueue):
-
     def __init__(self):
-        super(CursorItems, self).__init__(self)
-
-    @inlineCallbacks
-    def wait_for_new_item(self):
-        """
-        Callback whenever a new item arrived.
-        It use the behavior of put.
-        Warning: It ignores QueueUnderflow / QueueOverflow mechanisms.
-        """
-        d = Deferred(canceller=self._cancelGet)
-        self.waiting.append(d)
-        item = yield d
-        self.pending.append(item)
-        returnValue(item)
```

Comment:
`wait_for_new_time` was only ever used in tests because it has a test for the underlying batch pre-fetching.  This shouldn't actually be exposed, and the test now uses the underlying structures directly.